### DISCUSSION TITLE
Somehow display to user whether hard links are being used or not

### DISCUSSION THREAD

**Discussion by LittleEndu** [ppy/osu#21828] [Ideas]
Watching koifishu right now and people on his stream seem to be confused whether hard links work or not (and what drive lazer is using)

**Comment by smoogipoo** [comment]
There is already text on the setup screen, but given it was missed multiple times I’d say it’s not indicative enough.

**Comment by peppy** [comment]
I'm still figuring out the whole UX of the first run dialog, and this is one thing that I already have noted down.

**Reply by LittleEndu** [reply]
I would also suggest something like #21908 so what ever UI gives the best experience is the only UI that exists.

**Comment by peppy** [comment]
I'd probably make the text green/red when they are being used, to highlight the fact.
